symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/model/ActivityInfo.java

```diff
@@ -27,4 +29,18 @@
   /** Description of the activity (can contain multiple lines) */
   private String description;
 
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) { return true; }
+    if (o == null || getClass() != o.getClass()) { return false; }
+    ActivityInfo that = (ActivityInfo) o;
+
+    return this.hashCode() == that.hashCode();
```

Comment:
mm that looks odd, I don't remember seeing equality being implemented based on hashcode given there is a chance for collision I guess
Intellij can generate it for you

Reply:
We don't need it anymore in AcitivityInfo, It will be removed in next commit